src/mtcp/mtcp_restart.c

```diff
@@ -861,6 +862,14 @@ static int read_one_memory_area(int fd)
     }
   }
 
+#ifdef FAST_RST_VIA_MMAP
+    //int should_mmap_ckpt_image = 1;
+    else if ( 1 /* should_mmap_ckpt_image */ && (area.flags & MAP_ANONYMOUS)) {
```

Comment:
So, we only mmap in ANONYMOUS regions. Why? Could you please explain that in a comment?